docs/getting-started.mdx

```diff
@@ -115,24 +111,51 @@ When you're finished running press `ctrl + c` to stop the run command.
 
 Now that you've implemented a basic API and tested that it works, you can deploy it to one of the major cloud platforms.
 
-The first step is to set up your credentials for the cloud provider. You can follow along in the [provider overview](/docs/reference/aws).
+The first step is to set up your credentials for the cloud provider.
+
+- [AWS](/docs/reference/aws)
+- [Azure](/docs/reference/azure)
+- [GCP](/docs/reference/gcp)
+
+You'll then need to create a stack - a collection of resources identified in your project which will be deployed.
```

Comment:
A stack is more a combination of a project and a deployment target (cloud, region, etc.).

Reply:
Revised and added a page for stacks.